resources/functionMap.php

```diff
@@ -1060,7 +1060,7 @@
 'componere\cast_by_ref' => ['Type', 'arg1'=>'', 'object'=>''],
 'confirm_pdo_ibm_compiled' => [''],
 'connection_aborted' => ['0|1'],
-'connection_status' => ['int'],
+'connection_status' => ['0|1|2|3'],
```

Comment:
Perhaps more accurate:
```suggestion
'connection_status' => ['int-mask<CONNECTION_NORMAL|CONNECTION_ABORTED|CONNECTION_TIMEOUT>'],
```

Reply:
i was not aware of the `int-mask` type, that's great